benchmarks/src/generated/main/grpc/io/grpc/benchmarks/proto/BenchmarkServiceGrpc.java

```diff
@@ -268,11 +280,21 @@ public void invoke(Req request, io.grpc.stub.StreamObserver<Resp> responseObserv
     }
   }
 
-  private static final class BenchmarkServiceDescriptorSupplier implements io.grpc.protobuf.ProtoFileDescriptorSupplier {
+  private static final class BenchmarkServiceDescriptorSupplier implements io.grpc.protobuf.ProtoDescriptorSupplier {
     @java.lang.Override
     public com.google.protobuf.Descriptors.FileDescriptor getFileDescriptor() {
       return io.grpc.benchmarks.proto.Services.getDescriptor();
```

Comment:
Hmm okay. The main reason I've done this is because in order to get service/method descriptors you need to pass index. I guess adding new static `SERVICE_IDX` like we have one for methods + set of helpers in `ProtoUtils` would work. I'll change this.